src/web/mjs/engine/Storage.mjs

```diff
@@ -448,7 +449,9 @@ export default class Storage {
             }
             if (Engine.Settings.chapterFormat.value === extensions.cbz) {
                 this._createDirectoryChain(this.path.dirname(output));
-                promise = this._saveChapterPagesCBZ(output, pageData)
+                let mangaName = chapter.manga.title;
+                let chapterName = chapter.title;
+                promise = this._saveChapterPagesCBZ(output, pageData, mangaName, chapterName)
```

Comment:
May use `chapter.manga.name` and `chapter.title` directly as parameter instead of intermediate variables `mangaName` and `chapterName`.